.prettierrc

```diff
@@ -1,5 +1,7 @@
 {
+  "printWidth": 90,
   "singleQuote": true,
-  "trailingComma": "all",
-  "arrowParens": "always"
+  "trailingComma": "es5",
+  "jsxBracketSameLine": true,
+  "jsxSingleQuote": true
```

Comment:
All of these were added by Gregor. I prefer just using defaults as much as possible.
Do the jsx rules actually apply to Vue? Is see double quotes in the HTML in this PR so I'm asking what impact does this have. Same for same line bracket. I'd again prefer the defaults here. Brackets for less line change and double quotes in HTML since it's what the Firefox inspector will make out of it anyway.

Reply:
I'll remove these.